ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -125,7 +125,42 @@ class LeftNavMenuViewController: UIViewController {
 
     private var statusCell = MSFListCellState()
 
-    private var leftNavMenuSections: [MSFListSectionState] {
+    private var leftNavMenuList = MSFList()
+
+    private var leftNavAccountViewHeightConstraint: NSLayoutConstraint?
+
+    private lazy var leftNavAccountView: UIView = {
+        let chevron = UIImageView(image: UIImage(named: "ic_fluent_ios_chevron_right_20_filled"))
+        chevron.tintColor = Colors.textPrimary
+
+        persona.state.presence = .available
```

Comment:
Can you store persona.state in a constant to use it below?

`let personaState = persona.state`